ddtrace/contrib/pytest/plugin.py

```diff
@@ -214,7 +214,8 @@ def _start_test_module_span(pytest_package_item=None, pytest_module_item=None):
     test_module_span.set_tag_str(test.FRAMEWORK_VERSION, pytest.__version__)
     test_module_span.set_tag_str(test.COMMAND, _get_pytest_command(item.config))
     test_module_span.set_tag_str(_EVENT_TYPE, _MODULE_TYPE)
-    test_module_span.set_tag_str(_SESSION_ID, str(test_session_span.span_id))
+    if test_session_span:
```

Comment:
There should not be a case in which `test_session_span` is not available here. https://github.com/DataDog/dd-trace-py/commit/ae7bab3af39943bfdcdefec6f30bfe168633d2d7 was meant to help ensure we always have the full session -> module -> suite -> test hierarchy.

Is this breaking in a unit test, or are there issues with it in real-world use?

Reply:
This should've been addressed in this new commit. `unittest` is no longer enabled if `pytest` is running as we support `unittest` with `pytest` out of the box.